pages/1.11/installing/ent/custom/configuration/configuration-parameters/index.md

```diff
@@ -576,6 +582,14 @@ The bridge-mode networking for UCR is identical to bridge mode networking for Do
 
 The only constraint in selecting an IPv4 subnet for `dcos_ucr_default_bridge_subnet` is that the subnet should not be used on the network to which the agents are connected. In other words, this subnet should be routeable from only within an agent.
 
+[enterprise]
+### feature_dcos_storage_enabled
+[/enterprise]
+
+Enables advanced storage features in Mesos including [CSI](https://github.com/container-storage-interface/spec) support, and support for pre-installed CSI device plugins. 
```

Comment:
s/Mesos/DC/OS/

Reply:
Enables advanced storage features in DC/OS including Mesos CSI support, and pre-installed CSI device plugins.